File: android-project/vwanm/src/main/java/com/codekul/vwanm/MainActivity.java
package com.codekul.vwanm;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void animate(View vw) {
        Animation anim = AnimationUtils
                .loadAnimation(this, R.anim.rotate);
        anim.setAnimationListener(new Animation.AnimationListener() {
            @Override
            public void onAnimationStart(Animation animation) {

            }

            @Override
            public void onAnimationEnd(Animation animation) {
                startActivity(new Intent(MainActivity.this, DashActivity.class));
            }

            @Override
            public void onAnimationRepeat(Animation animation) {

            }
        });
        findViewById(R.id.img).startAnimation(anim);
    }
}

File: android-project/cmpdvw/src/main/java/com/codekul/cmpdvw/MainActivity.kt
package com.codekul.cmpdvw

import android.content.Context
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.LayoutInflater
import android.widget.ImageView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    //lateinit var infalter : LayoutInflater

    val inflater: LayoutInflater by lazy {
        getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btOk.setOnClickListener {
            val vw = inflater
                    .inflate(
                            R.layout.mixed_view,
                            ll,
                            false
                    )
            ll.addView(vw)

            vw.findViewById<ImageView>(R.id.img).setOnClickListener {
                (it as ImageView).setImageResource(R.drawable.abc_btn_switch_to_on_mtrl_00012)
            }
//            vw.setOnClickListener {
//                Log.i("@codekul","All View clicked")
//            }
        }
    }
}
